src/main/java/ch/njol/skript/conditions/CondIsSaddled.java

```diff
@@ -0,0 +1,51 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.Material;
+import org.bukkit.entity.AbstractHorse;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Steerable;
+import org.bukkit.inventory.ItemStack;
+
+@Name("Is Saddled")
+@Description("Checks whether a given entity (horse or steerable) is wearing a saddle.")
+@Examples("send whether {_horse} is saddled")
+@Since("INSERT VERSION")
+public class CondIsSaddled extends PropertyCondition<LivingEntity> {
+
+	static {
+		register(CondIsSaddled.class, "[:properly] saddled", "livingentities");
```

Comment:
What does properly saddled mean? Maybe point it out in the description?

Reply:
Properly saddled is for whether the item set as the saddle is actually a saddle. Because the `setSaddle` method takes any ItemStack and `hasSaddle` just returns true if the saddle item is set at all (i.e. not necessarily to a saddle)